_includes/head.html

```diff
@@ -1,11 +1,19 @@
 <head>
-  <link href="http://gmpg.org/xfn/11" rel="profile">
+  <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-
-  <!-- Enable responsiveness on mobile devices-->
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
 
+  <link href="http://gmpg.org/xfn/11" rel="profile">
+
+  <meta name="description" content="Lessons about the Elixir programming language, inspired by Twitter’s Scala School.">
+  <meta name="author" content="Sean Callan">
+
+  <meta property="og:title" content="Elixir School - Lessons about the Elixir programming language">
+  <meta property="og:image" content="http://elixir-lang.org/images/logo/logo.png">
+  <meta property="og:description" content="Lessons about the Elixir programming language, inspired by Twitter’s Scala School.">
+
+  <meta name="google-site-verification" content="6ZHfETSmHjwNySjadWFTG-NUsVD_Lm2T46jdmYy43Zg" />
```

Comment:
@psantos10 I already setup Elixir School in the Webmaster Tools, using the Domain name provider method.  I'd be glad to add you as a user if you could share your email address.